.github/workflows/test-containers.yaml

```
@@ -0,0 +1,126 @@
+    name: "${{ matrix.image-type }}"
+    needs: find-images
+    runs-on: ubuntu-24.04
+    strategy:
+      fail-fast: false
+      matrix:
+        image-type:
+          - jupyter-minimal-ubi9-python-3.12
+          - jupyter-datascience-ubi9-python-3.12
+          - runtime-datascience-ubi9-python-3.12
+    env:
+      TEST_IMAGE: "${{ env.IMAGE_REGISTRY }}:${{ matrix.image-type }}-main_${{ needs.find-images.outputs.image-sha }}_odh_linux_amd64"
```

actionlint (1.7.12)

[error] context "env" is not allowed here. available contexts are "github", "inputs", "matrix", "needs", "secrets", "strategy", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)